styles/main/_mapview.scss

```diff
@@ -5,16 +5,12 @@
 	left: 0;
 	width: 100%;
 	height: 100%;
-	background-color: rgba(100, 10, 10, 0.98);
+	background-color: #0f0f0f;
 	transition: background-color 0.3s;
 
 	// Modes -------------------------------------------------------------- //
-	&.view {
-		background-color: inherit;
-	}
 
 	&.full {
-		background-color: black(1);
```

Comment:
See my comment for `_imageview.scss`.

Reply:
In order to keep the discussion at one place, I will close this conversation here. We should continue the discussion in  https://github.com/LycheeOrg/Lychee-front/pull/323#discussion_r1001350913.